rust/src/crypto/encrypt.rs

```diff
@@ -59,37 +66,40 @@ impl ByteObject for PublicEncryptKey {
 }
 
 impl PublicEncryptKey {
-    /// Length in bytes of a [`PublicEncryptKey`]
+    /// Length in bytes of this public key.
     pub const LENGTH: usize = box_::PUBLICKEYBYTES;
 
-    /// Encrypt a message `m` with this public key. The resulting
-    /// ciphertext length is [`SEALBYTES`] + `m.len()`.
+    /// Encrypts a message `m` with this public key. The resulting ciphertext length is
+    /// [`SEALBYTES`]` + m.len()`.
     ///
-    /// The function creates a new key pair for each message, and
-    /// attaches the public key to the ciphertext. The secret key is
-    /// overwritten and is not accessible after this function returns.
+    /// The function creates a new ephemeral key pair for each message, and attaches the ephemeral
```

Comment:
Does `m` contains more than one message? If not I would change it to:
`The function creates a new ephemeral key pair for a message`

Reply:
i'll change it. it's one message per returned `Vec<u8>`.

Reply:
yeah i agree the "each message" part is a bit confusing. you could even say "**the** message", or just `m`